apps/src/blockly/addons/cdoVerticalFlyout.js

```diff
@@ -29,11 +29,21 @@ export default class VerticalFlyout extends GoogleBlockly.VerticalFlyout {
     flyoutWidth += Blockly.Scrollbar.scrollbarThickness;
 
     /* Begin CDO Customization */
-    // Constrain the flyout to not be wider than 40% of the total workspace space.
-    flyoutWidth = Math.min(
-      flyoutWidth,
-      this.targetWorkspace.getMetricsManager().getSvgMetrics().width * 0.4
-    );
+    const isMainWorkspace =
+      this.targetWorkspace.id === Blockly.getMainWorkspace()?.id;
+    if (isMainWorkspace) {
+      // Constrain the main workspace flyout to not be wider than 40% of the total workspace space.
+      flyoutWidth = Math.min(
+        flyoutWidth,
+        this.targetWorkspace.getMetricsManager().getSvgMetrics().width * 0.4
+      );
+    } else {
+      if (Blockly.getMainWorkspace()) {
```

Comment:
nit: `else if`?

Reply:
Haha, yup. Updated.

Reply:
Yes, and that seems much clearer. Will update!